cpp/tests/test_odeseir.cpp

```diff
@@ -25,6 +25,18 @@
 #include "memilio/compartments/simulation.h"
 #include <gtest/gtest.h>
 
+TEST(TestSeir, simulateDefault)
```

Comment:
Shouldn't it be TestOdeSeir?

Reply:
Yes. but everywhere in the file. Should be changed with some future commit